src/main/java/hsleiden/stenentijdperk/stenentijdperk/Models/PlayerModel.java

```diff
@@ -51,6 +53,8 @@ public int getPunten() {
 
     public void setPunten(int punten) {
         this.punten = punten;
+        if (this.getPlayerNumber() != null)
+            FirebaseController.updateDocument(String.valueOf(this.getLobby()), this.getPlayerNumber(), "punten", this.punten);
```

Comment:
Voor deze en alle volgende updates. Het is goed, maar erg intensief op de request naar firebase.